yb-voyager/cmd/metadataDB.go

```diff
@@ -65,6 +72,8 @@ func initMetaDB(path string) error {
 	}
 	cmds := []string{
 		fmt.Sprintf(`CREATE TABLE %s (segment_no INTEGER PRIMARY KEY, file_path TEXT, size_committed INTEGER );`, QUEUE_SEGMENT_META_TABLE_NAME),
+		fmt.Sprintf(`CREATE TABLE %s (run_id TEXT, timestamp_minute INTEGER, num_total INTEGER, num_inserts INTEGER, num_updates INTEGER, num_deletes INTEGER, PRIMARY KEY(run_id, timestamp_minute) );`, EXPORTED_EVENTS_STATS_TABLE_NAME),
```

Comment:
You can define these templates as top level constants in this file. Then you can make the DDL statements more readable with proper indentation.